### ISSUE TITLE
Target cool termperature equals cached state

### ISSUE THREAD

**Issue by kb1ibt** [sman591/homebridge-lg-thinq-ac#12]
**LG Air Conditioner Model**
LW8017ERSM

**Describe the bug**
When I am trying to change the AC cooling temperature it is not actually changing via HomeKit/HomeBridge but it is working via LG ThinQ App.

**Symptom check**
* **Does your unit appear in the Home app?** Yes
* **Is the current on/off state, temperature, and mode being reported?** Yes
* **Does your unit respond to the controls?** Some, it responds to changing fan speeds but not temperature changes.

**Relevant logs**
```[5/29/2020, 15:05:50] [LgThinqAirConditioner] Triggered SET Active: 1
[5/29/2020, 15:05:50] [LgThinqAirConditioner] Power state equals cached state. Skipping. on
[5/29/2020, 15:05:50] [LgThinqAirConditioner] Triggered SET Heater Cooler State: 2
[5/29/2020, 15:05:50] [LgThinqAirConditioner] Target heater cooler state equals cached state. Skipping. cool
[5/29/2020, 15:05:51] [LgThinqAirConditioner] Triggered SET Cooling Threshold Temperature: 20
[5/29/2020, 15:05:51] [LgThinqAirConditioner] Target cool termperature equals cached state. Skipping. 20
[5/29/2020, 15:05:51] [LgThinqAirConditioner] Triggered SET Rotation Speed: 66
[5/29/2020, 15:05:51] [LgThinqAirConditioner] Fan state equals cached state. Skipping. medium
[5/29/2020, 15:05:51] [LgThinqAirConditioner] Triggered SET Heating Threshold Temperature: 0
[5/29/2020, 15:05:51] [LgThinqAirConditioner] Target heat termperature equals cached state. Skipping. 0
```

**Comment by kb1ibt** [comment]
`[5/29/2020, 15:13:28] [LgThinqAirConditioner] device response {
  resultCode: '0000',
  result: {
    appType: 'NUTS',
    modelCountryCode: 'WW',
    countryCode: 'US',
    modelName: 'WIN_056905_WW',
    deviceType: 401,
    deviceCode: 'AI01',
    alias: 'Bedroom AC',
    deviceId: '85ef32bb-e5a5-1067-a4f4-44cb8b63bad5',
    fwVer: '',
    imageFileName: 'ac_home_window_airconditioner_img.png',
    ssid: 'Westeros',
    softapId: '',
    softapPass: '',
    macAddress: '44:cb:8b:63:ba:d5',
    networkType: '02',
    timezoneCode: 'America/New_York',
    timezoneCodeAlias: 'USA/New_York',
    utcOffset: -5,
    utcOffsetDisplay: '-05:00',
    dstOffset: -4,
    dstOffsetDisplay: '-04:00',
    curOffset: -4,
    curOffsetDisplay: '-04:00',
    sdsGuide: '{"deviceCode":"AI01"}',
    newRegYn: 'N',
    remoteControlType: '',
    userNo: 'US2005070127231',
    tftYn: 'N',
    deviceState: 'E',
    snapshot: {
      'airState.windStrength': 4,
      static: [Object],
      'airState.tempState.current': 21.5,
      'airState.reservation.sleepTime': 0,
      'airState.filterMngStates.maxTime': 250,
      'airState.reservation.targetTimeToStart': 0,
      'airState.operation': 1,
      'airState.opMode': 0,
      'airState.quality.sensorMon': 0,
      'airState.filterMngStates.useTime': 203,
      meta: [Object],
      online: true,
      timestamp: 1590779604821,
      'airState.reservation.targetTimeToStop': 0,
      'airState.tempState.target': 22,
      mid: 171625320,
      'airState.diagCode': 0,
      'airState.wDir.leftRight': 0
    },
    online: true,
    platformType: 'thinq2',
    area: 334505,
    regDt: 20200527161838,
    blackboxYn: 'Y',
    modelProtocol: 'STANDARD',
    receipeVersion: 0,
    activeSaving: 'OFF',
    smartCareV2: 'OFF',
    order: 0,
    drServiceYn: 'N',
    regDtUtc: '20200527201838',
    groupableYn: 'Y',
    controllableYn: 'Y',
    combinedProductYn: 'N',
    masterYn: 'Y',
    tclcount: 0
  }
}
[5/29/2020, 15:13:28] [LgThinqAirConditioner] device response.result.snapshot {
  'airState.windStrength': 4,
  static: { deviceType: '401', countryCode: 'US' },
  'airState.tempState.current': 21.5,
  'airState.reservation.sleepTime': 0,
  'airState.filterMngStates.maxTime': 250,
  'airState.reservation.targetTimeToStart': 0,
  'airState.operation': 1,
  'airState.opMode': 0,
  'airState.quality.sensorMon': 0,
  'airState.filterMngStates.useTime': 203,
  meta: { allDeviceInfoUpdate: false, messageId: 'u2XOJS42SR26TZ_f0HxxuQ' },
  online: true,
  timestamp: 1590779604821,
  'airState.reservation.targetTimeToStop': 0,
  'airState.tempState.target': 22,
  mid: 171625320,
  'airState.diagCode': 0,
  'airState.wDir.leftRight': 0
}
[5/29/2020, 15:13:28] [LgThinqAirConditioner] Pushed updates to HomeKit {
  power: 'on',
  currentTemperature: 21.5,
  targetTemperatureCool: 22,
  targetTemperatureHeat: null,
  mode: 'cool',
  fan: 'medium'
}
`

**Comment by sman591** [comment]
Interesting... let me look into on what’s happening here, thanks for reporting!

**Comment by kb1ibt** [comment]
So my wife accidentally demonstrated that it is controlling the temperature, but only in one direction, up. I was showing her what I was starting to implement, and she thought oh let’s see what this does and spun the temperature from 72F to 81F and I heard the beep that it took the new setting; but when I tried to lower it back down, it again threw the cached state. So I had to switch over to the ThinQ app to bring the temp back to normal.
